forgotPass.php

```diff
@@ -0,0 +1,59 @@
+<?php
+require("model/DbConnection.php");
+require("model/Features.php");
+
+// If already loggedin redirect to submit page.
+if (isset($_SESSION['loggedIn']) && $_SESSION['loggedIn'] == TRUE)
+  header("Location:index.php");
+
+// Connect with Credential table database.
+$db = new DbConnection();
+
+// Checks first if submit is clicked
+if (isset($_POST["mailId"])) {
+
+  // If id and pass fields are not empty.
+  if ($_POST["mailId"] != "") {
+    if (!preg_match('/^([a-z0-9\+_\-]+)(\.[a-z0-9\+_\-]+)*@([a-z0-9\-]+\.)+[a-z]{2,6}$/ix', $_POST['mailId'])) { ?>
+      <h4 class='error br'>Invalid email address.</h4>
+    <?php }
```

Comment:
You have already created method to validate email pattern in Model file then why are you not calling that method here?